Ports/nesalizer/patches/0003-Hard-code-relative-path-for-SDL2-headers-instead-of-.patch

```diff
@@ -0,0 +1,35 @@
+From 1359bc479c2c90c83332edec8b8533c04571db3f Mon Sep 17 00:00:00 2001
+From: Dan MacDonald <allcoms@gmail.com>
+Date: Mon, 9 Dec 2019 02:14:37 +0000
+Subject: [PATCH 3/6] Hard-code relative path for SDL2 headers instead of using
+ sdl2-config
+
+---
+ Makefile | 4 ++--
+ 1 file changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/Makefile b/Makefile
+index 60563bf..822ba70 100644
+--- a/Makefile
++++ b/Makefile
+@@ -133,7 +133,7 @@ endif
+ 
+ # _FILE_OFFSET_BITS=64 gives nicer errors for large files (even though we don't
+ # support them on 32-bit systems)
+-compile_flags += $(warnings) -D_FILE_OFFSET_BITS=64 $(shell sdl2-config --cflags)
++compile_flags += $(warnings) -D_FILE_OFFSET_BITS=64 -I../../SDL2/SDL-master-serenity/include
```

Comment:
This is bad. Please use the (proper) installed location of the headers instead of the development tree. But in theory, this should be handled by the default include paths combined with `--sysroot` anyways.

Since this patch seems to have originated from SerenityPorts like this, keep it as is and fix it up in a seperate commit.

Reply:
Also, I'm pretty sure the patch in its current form is broken anyways, since the SDL2 branch has since been renamed to `main-serenity`. If it builds like this, we can probably just remove that include directory completely since we install SDL in a very standard location.

Reply:
The patch did not do anything as it picks up SDL2 using the `makeopts`.

I just removed the patch to keep things clean. Do you think adding a revert patch would be better?

Reply:
Based on Linusgs answer we should keep the patches as small as possible.